src/libcollectionstest/str.rs

```diff
@@ -468,6 +468,18 @@ fn test_is_utf8() {
     assert!(from_utf8(&[0xEF, 0xBF, 0xBF]).is_ok());
     assert!(from_utf8(&[0xF0, 0x90, 0x80, 0x80]).is_ok());
     assert!(from_utf8(&[0xF4, 0x8F, 0xBF, 0xBF]).is_ok());
+
+    // deny embedded in long stretches of ascii
```

Comment:
I don't really know what this specific set of tests is doing.

I always have a bit of a sad when there are these giant "test everything" tests; my personal pref would be another test like `is_utf8_is_not_tricked_by_non_ascii_in_long_stretches_of_ascii`. No need to add `test_`, no need to have a comment, a failed test tells you what failed. :smile_cat: